about.html

```diff
@@ -0,0 +1,112 @@
+          <li><a class="menu-link" href="#">News</a></li>
+        </ul>
+        <button class="btn-campain"><a href="#">CMC World</a></button>
+      </nav>
+    </div>      
+  </header>
+  <div class="page">
+    <section class="principal">
+      <div class="tittle">
+        <h2 class="txt-tit-about">"United by the same Faith"</h2>
+        <h1 class="h1-about">CATHOLIC MUSIC CONCERT<br>WORLD TOUR July 2023</h1>
+        <p class="desc-about">A musical meeting of prayer and adoration with the best exponents of Catholic music. Three nights full of blessings, accompanied by teachings that will help us grow in our Faith.</p>
+        <div class="desc-email">
+          <p class="contact-about">Please contact us per Email for any futher questions about CMC World Tour 2023</p>
+          <a class="email" href="email://cmcworld2023@catholic.org">cmcworld2023@catholic.org</a> 
+        </div>        
+      </div>
+    </section>
+    <section class="logo">
+      <div class="logo-container">
+        <h2 class="tit-about">CATHOLIC MUSIC CONCERT<br>WORLD TOUR July 2023</h2>
+        <hr class="small-line">
+        <p class="logo-desc">The logo of CMC World Tour 2023 was decided trough the logo competition from 20.October to 22.September.</p>
+        <img class="logo-big" src="images/logo-big.png" alt="Logo CMC 2023">
+      </div>
+    </section>
+    <section class="past-events">
+      <div class="past-container">
+        <h2 class="tit-about">See the past CMC World Tour</h2>
+        <hr class="small-line">
+        <p class="logo-desc">Take a look at the two CMC World Tour wich tookin South America.</p>
+        <div class="past-img">
+          <img src="images/MCM2022.png" alt="Photo of the MCM 2022" class="past">
+        <img src="images/mcm2015.png" alt="Photo of the MCM 2015" class="past">
+        </div>          
+      </div>
+    </section>
+    <footer>
+      <div class="partner">
+        <h2>Partner</h2>
+        <hr class="small-line">
+        <div class="logos">
+          <a href="https://www.mozilla.org/es-ES/firefox/new/"><img class="logo-partner" src="images/log-moz.png" alt="Mozilla Logo"></a>
+          <a href="https://www.google.com/"><img class="logo-partner" src="images/log-google.png" alt="Google Logo"></a>
+          <a href="https://www.naver.com/"><img class="logo-partner" src="images/log-naver.png" alt="Naver Logo"></a>
+          <a href="https://www.kakaocorp.com/page/"><img class="logo-partner" src="images/log-daum.png" alt="Daumkakao Logo"></a>
+          <a href="https://www.airbnb.com/"><img class="logo-partner" src="images/log-airbnb.png" alt="Airbnb Logo"></a>          
+        </div>
+      </div>
+      <div class="copyright">
+        <img src="images/logo-sinodo.png" alt="Logo">
+        <div class="cop-desc">
+          <p class="cop-text">2023 - Synod 21-24<br>Concert Music Catholic</p>
+          <p class="cop-text">Some Rights Reserved</p>
+        </div>
+      </div>
```

Comment:
in the footer section in your about page your items should be more centralized and your  logo be further apart from your text to avoid clustering and poor UX

your design 
![image](https://user-images.githubusercontent.com/60612329/227384437-b8705b66-acdb-48ba-8524-c2430eb72355.png)

template design 
![image](https://user-images.githubusercontent.com/60612329/227384568-a636ac86-89a4-4284-b26a-e9d546d4516c.png)

Reply:
@alexander16108, I corrected:

![image](https://user-images.githubusercontent.com/110146573/227437825-42cbd396-0129-41ba-b6f1-c91d4344cd49.png)